.github/actions/next-stats-action/src/prepare/repo-setup.js

```diff
@@ -124,18 +125,11 @@ module.exports = (actionInfo) => {
               pkgData.dependencies['@next/swc'] =
                 pkgDatas.get('@next/swc').packedPkgPath
             } else {
-              pkgData.files.push('native/*')
+              pkgData.files.push('native')
             }
           }
         }
 
-        if (pkgData?.scripts?.prepublishOnly) {
-          // There's a bug in `pnpm pack` where it will run
-          // the prepublishOnly script and that will fail.
-          // See https://github.com/pnpm/pnpm/issues/2941
-          delete pkgData.scripts.prepublishOnly
-        }
```

Comment:
I think we still need this since the linked issue was fixed in `pnpm@7.26.0` but we are using `pnpm@7.24.3` above

Reply:
We're not using `pnpm` anymore, `yarn pack` just works so we should just keep using it 🤷

Reply:
@ijjk This is only used for gh actions, it wont be used when I run `pnpm test-start test/e2e/foo` right?